webapp/channels/src/components/search_bar/search_bar.tsx

```diff
@@ -155,7 +155,7 @@ const SearchBar: React.FunctionComponent<Props> = (props: Props): JSX.Element =>
                     ref={getSearch}
                     id={props.isSideBarRight ? 'sbrSearchBox' : 'searchBox'}
                     tabIndex='0'
-                    className={'search-bar form-control a11y__region'}
+                    className={'search-bar form-control'}
```

Comment:
Why is this being removed from the old search?

Reply:
I don't know, let me get rid of the change.